### DISCUSSION TITLE
Scroll does not start with arrow down key after page load

### DISCUSSION THREAD

**Discussion by ashehzadgec** [idiotWu/smooth-scrollbar#336] [Q&A] [answered]
The problem is whenever page is loaded and then I have to click through mouse on page or use mouse wheel to scroll the page. Otherwise page will not scroll e.g(after page loading when I press arrow down key, page should scroll but it doesn't). This issue is persistent on all modern browsers like chrome, Mozilla, opera and edge.
- Browser:
- Version of smooth-scrollbar:
8
## Issue Summary

On page load, scroll does not work by using arrow down key. All modern browsers. 
## Current Behavior

On page load, scroll should work by using arrow down key but it doesn't.

## Expected Behavior
On page load, scroll should work by using arrow down key.

On page load, scroll should work by using arrow down key.

**Comment by dolphin-wood** [comment, marked as answer]
The scrollable area requires to be focused to enable keyboard scrolling. So in your case, you should call `container.focus()` after page loaded.

**Comment by ashehzadgec** [comment]
Hey Dolphin

Thanks a lot for your help. It worked.

**Comment by kiran-kadam911** [comment]
@idiotWu  can you pls share an example. Actually, I was trying this thing with container class for example `$('.scroller').focus();` but no luck on window.load
